presubmit-service/check_if_pr_can_cherry_pick.sh

```diff
@@ -0,0 +1,63 @@
+#! /bin/bash
+
+# The organization containing the repo to be checked against, e.g. angular
+ORG=$1
+# The repo to be checked against, e.g. material2
+REPO=$2
+# The PR number to retrieve the patch files for, e.g. 1234
+PR=$3;
+# The branch in the repo to check for cherry-picking onto, e.g. 6.1.x
+BRANCH=$4;
+# The fullname of the repository to be checked against, e.g. angular/material2
+FULL_REPO="$ORG/$REPO"
+# Whether the PR can be cherry-picked on to the targetted branch.
+RESULT=false;
+
+# If the the repos directory does not exist, create it.
+[ ! -d "local_repo" ] && mkdir local_repo;
+cd local_repo;
```

Comment:
Shouldn't this live in `/tmp` as well? Something like `/tmp/presubmit-service/git-repository-pool`? (or a better name)

Reply:
Definitely a fair point. Usually on linux I think you'd store something like that into `opt/` then, but I guess somewhere in `$HOME` is fine as well. Maybe `$HOME/.presubmit-service`?